openml/flows/sklearn_converter.py

```diff
@@ -625,28 +625,33 @@ def _serialize_cross_validator(o):
 
 
 def _check_n_jobs(model):
```

Comment:
Given the additional duties of this function, could you please change the function name?

Reply:
There are no additional duties, just a refactoring (to make subfunction more readible) and the correct assumptions of the param_grid (which can be of datatype dict and list of dicts)